packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st

```diff
@@ -0,0 +1,8 @@
+constants
+userStoryString
+	
+	^ 'AC Calculator User Story
+	   In order to test my sophisticated calculator class
+	   As a developer
+	   I want to do some complicated calculations'
+	"self userStoryName, String cr, self userStoryContent"
```

Comment:
why dont we use the construction in line 8? The hardcoded version could lead to consistancy problems when changing US name and content in their getter methods

Reply:
yes we just forgot about that, thank you.